bc_obps/reporting/migrations/0027_storage_of_petroleum_products.py

```diff
@@ -0,0 +1,261 @@
+# Generated by Django 5.0.9 on 2023-10-21 22:20
+
+from django.db import migrations
+import os
+import json
+
+
+def init_additional_methodology_data(apps, schema_monitor):
```

Comment:
Nice job on the activity page. 
I just have a suggestion,
 I would include this initialization directly within the existing init_methodology_data function(migrations/prod_data). Since all source types, activities, and methodologies are already being initialized there.